demo/index.html

```diff
@@ -206,29 +206,33 @@
         </div>
       </template>
       <script>
-        Polymer({
-          is: 'demo-page',
-
-          preload: function(e) {
-            var img = document.querySelector('#' + e.target.getAttribute('target'));
-            img.src = './polymer.svg?' + Math.random();
-            e.target.textContent = 'Reload image';
-          },
-
-          _computeLogoAltText: function(loading, loaded) {
-            var text = 'The Polymer logo. ';
-
-            if (loading) {
-              text += '(loading)';
-            } else if (loaded) {
-              text += '(loaded)';
-            } else {
-              text += '(not loaded)';
-            }
 
-            return text;
-          }
+        window.addEventListener('WebComponentsReady', function() {
```

Comment:
@keanulee should this use https://www.polymer-project.org/1.0/docs/devguide/registering-elements.html#main-document-definitions instead? Or `Polymer.whenReady`?